config/crd/bases/keda.sh_scaledjobs.yaml

```diff
@@ -7401,6 +7401,11 @@ spec:
                       additionalProperties:
                         type: string
                       type: object
+                    metricType:
```

Comment:
`ScaleJob` CRD shouldn't be changed, we need to find a way how to not add it here.

Reply:
Any idea how to go about this? I tried to dive into this but couldn't find an elegant way given how `ScaleTriggers` is shared between `ScaledJobSpec` and `ScaledObjectSpec`

Reply:
@coderanger by chance, don't you have any suggestion how to fix this?